### ISSUE TITLE
Empty needle in Utility.php

### ISSUE THREAD

**Issue by pawelkmpt** [koolphp/koolreport#21]
I just updated composer package to latest version (3.25.0) and I see following warning:

```
PHP Warning:  strpos(): Empty needle in /vendor/koolphp/koolreport/src/core/Utility.php on line 493
```

**Comment by lkraav** [comment]
I think solved in 3.25.1 and https://www.koolreport.com/forum/topics/669

Maybe @koolphp can confirm?

**Comment by koolphp** [comment]
Confirmed!
